src/hooks/useNotificationTypes.tsx

```diff
@@ -210,59 +211,96 @@ export const notificationTypes: NotificationTypeConfig[] = [
         // eslint-disable-next-line no-restricted-syntax
         for (const transfer of transferNotifications) {
-          const { fromChainId, status, txHash, toAmount, type, isExchange } = transfer;
-          const isFinished =
-            (Boolean(status) && status?.squidTransactionStatus !== 'ongoing') || isExchange;
-          const icon = <Icon iconName={isFinished ? IconName.Transfer : IconName.Clock} />;
+          const { fromChainId, status, txHash, toAmount, type, isExchange, depositSubaccount } =
+            transfer;
 
           const transferType =
             type ??
             (fromChainId === selectedDydxChainId
               ? TransferNotificationTypes.Withdrawal
               : TransferNotificationTypes.Deposit);
 
-          const title = stringGetter({
-            key: {
-              deposit: isFinished ? STRING_KEYS.DEPOSIT : STRING_KEYS.DEPOSIT_IN_PROGRESS,
-              withdrawal: isFinished ? STRING_KEYS.WITHDRAW : STRING_KEYS.WITHDRAW_IN_PROGRESS,
-            }[transferType],
-          });
-
-          const toChainEta = status?.toChain?.chainData?.estimatedRouteDuration ?? 0;
-          // TODO: remove typeguards once skip implements estimatedrouteduration
-          // https://linear.app/dydx/issue/OTE-475/[web]-migration-followup-estimatedrouteduration
-          const estimatedDuration =
-            typeof toChainEta === 'string' ? toChainEta : formatSeconds(Math.max(toChainEta, 0));
-          const body = stringGetter({
-            key: STRING_KEYS.DEPOSIT_STATUS,
-            params: {
-              AMOUNT_USD: `${toAmount} ${DydxChainAsset.USDC.toUpperCase()}`,
-              ESTIMATED_DURATION: estimatedDuration,
-            },
-          });
-
-          trigger(
-            txHash,
-            {
-              icon,
-              title,
-              body,
-              renderCustomBody: ({ isToast, notification }) => (
-                <TransferStatusNotification
-                  isToast={isToast}
-                  slotIcon={icon}
-                  slotTitle={title}
-                  transfer={transfer}
-                  type={transferType}
-                  triggeredAt={transfer.triggeredAt}
-                  notification={notification}
-                />
-              ),
-              toastSensitivity: 'foreground',
-              groupKey: NotificationType.SquidTransfer,
-            },
-            [isFinished]
-          );
+          const nobleChainId = getNobleChainId();
+          const isCosmosTransfer = [nobleChainId].includes(fromChainId ?? '');
+          if (isCosmosTransfer) {
+            const icon = <$AssetIcon symbol="USDC" />;
+            const isFinished = depositSubaccount?.txHash && !depositSubaccount?.needToDeposit;
+            const title = isFinished
+              ? stringGetter({
+                  key: STRING_KEYS.DEPOSIT,
+                })
+              : // TODO: Need to add Localization
+                'Confirm Pending Deposit';
```

Comment:
The phrases that need to be added to v4-localization are marked with the comment 
`// TODO: Need to add Localization`